tests/fixture/tmpnet/network.go

```diff
@@ -917,3 +934,20 @@ func GetReusableNetworkPathForOwner(owner string) (string, error) {
 	}
 	return filepath.Join(networkPath, "latest_"+owner), nil
 }
+
+func checkVMBinariesExist(subnets []*Subnet, pluginDir string) error {
+	errs := []error{}
+	for _, subnet := range subnets {
+		for _, chain := range subnet.Chains {
+			pluginPath := filepath.Join(pluginDir, chain.VMID.String())
+			_, err := os.Stat(pluginPath)
+			if err != nil {
```

Comment:
Should this be checking `os.ErrNotExist` and bubbling the error up otherwise instead?